pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -74,7 +74,7 @@ func GenerateKubeadmYAML(cc config.ClusterConfig, n config.Node, r cruntime.Mana
 		return nil, errors.Wrap(err, "generating extra component config for kubeadm")
 	}
 
-	cnm, err := cni.New(cc)
+	cnm, err := cni.New(&cc)
```

Comment:
what was the reason for this ?

Reply:
this is to be able to set kubelet.cni-conf-dir via cc.KubernetesConfig.ExtraOptions in cni.configureCNI if needed